.devcontainer/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV BUILD_DIR=/build
 ENV ENVOY_STDLIB=libc++
 
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get -y update \
+RUN apt-get -y update --allow-insecure-repositories \
```

Comment:
Which repo is insecure now?  I wonder if we should leave a comment here mentioning which one it is, so that we can periodically review this and attempt to remove this flag in the future.

Reply:
I tracked this down to https://github.com/10gen/envoy-serverless/pull/132. @ptravers do you know what the insecure repo was that you were referring to in the PR?